packages/menu/src/Menu.tsx

```diff
@@ -39,15 +40,15 @@ interface MenuProps
    * Callback to change the open state of the Menu.
    *
    */
-  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  setOpen?: React.Dispatch<React.SetStateAction<boolean>>;
 
   /**
    * Callback to determine whether or not Menu should close when user tries to close it.
    *
    */
   shouldClose?: () => boolean;
 
-  children: React.ReactNode;
+  children: Array<React.ReactElement>;
```

Comment:
Hmm... Isn't there a type for children specifically that includes text nodes, arrays, and react elements?